LEGO1/mxsoundmanager.h

```diff
@@ -1,9 +1,14 @@
 #ifndef MXSOUNDMANAGER_H
 #define MXSOUNDMANAGER_H
 
-class MxSoundManager
+#include "mxcore.h"
+
+class MxSoundManager : public MxCore
```

Comment:
This inheritance is not correct.
There are 2 levels of inheritance between `MxSoundManager` and `MxCore`.